### PR TITLE
PLANET-5646 Remove the top bar and move the search to the sidebar

### PR DESCRIPTION
**Pull request by mleray**
### Description

See https://jira.greenpeace.org/browse/PLANET-5646

### Testing

You can test it out on local by cloning the handbook's [child-theme](https://github.com/greenpeace/planet4-child-theme-handbook) into your `persistence/app/public/wp-content/themes/` folder and switching to this branch. Then you can turn the theme on in the backend in `Appearance > Themes` 🙂 

Alternatively, you can check out the handbook's [dev site](https://planet4-dev.greenpeace.org/handbook/) where the changes have been deployed!

### CODE REVIEW

File: css/navigation.css [hunk 1]
```diff
@@ -66,21 +57,28 @@
...
+    padding: 5px 10px;
   }
 
-  .top-nav-search-btn {
-    color: var(--light-grey);
-    background-color: var(--sidebar-blue) !important;
+  .nav-search-btn {
+    position: absolute;
+    border: none;
+    border: none;
```
Comment (GP-Dan-Tovbein) [inline css/navigation.css:75]: Duplicated attribute

File: css/navigation.css [hunk 2]
```diff
@@ -66,21 +57,28 @@
...
+    position: absolute;
+    font-size: 1.25rem;
```
Comment (GP-Dan-Tovbein) [inline css/navigation.css:77]: We're using `px` unit for all. So, I'm not sure why in this case we should use `rem`.

**Review by GP-Dan-Tovbein — CHANGES_REQUESTED** [review]
Code looks good to me but I left a couple of comments.

File: css/navigation.css (hunk 2 above)
Comment (GP-Dan-Tovbein) [reply css/navigation.css:77]: You can use also `$font-size-md`

File: css/navigation.css (hunk 1 above)
Comment (mleray) [reply css/navigation.css]: Removed! 👍

File: style.css [hunk 3]
```diff
@@ -13,14 +13,94 @@ Text Domain: planet4-child-theme-handbook
...
+    width: var(--sidebar-width);
+    position: absolute;
+    left: 0;
+    bottom: 0;
+    top: 0;
+    background-color: var(--sidebar-blue);
+    color: var(--white);
+    padding: 10px 20px;
+    z-index: 99;
+    font-family: Roboto;
```
Comment (GP-Dan-Tovbein) [inline style.css:42]: You can change to `font-family: Roboto, sans-serif;` like [here](https://github.com/greenpeace/planet4-child-theme-handbook/blob/master/style.css#L35)

Comment (GP-Dan-Tovbein) [reply style.css:42]: Also could be fine if we move this into a variable, i.e. `$roboto: Roboto, sans-serif;`

**Review by GP-Dan-Tovbein — APPROVED** [review]
I left some suggestions but the code looks good!

**Pull request merged**